docs/integrations/preimum_packs.md

```diff
@@ -0,0 +1,20 @@
+---
+id: premium_packs
+title: Premium Packs setup
+---
+
+In this document we will go over the entire process for developing paid content for XSOAR’s marketplace, step by step.
+
+1. Duplicate the demisto/content-external-template repository by clicking on the “Use this template”, then choose to create it as a private repository under your user.  
+<img src="../doc_imgs/integrations/demisto:content-external-template.png" width="480"></img>  
```

Comment:
The image seems very small. And it is not clickable. Increase the image size and see how/can we make it clickable (in order to see full image)

Reply:
Updated the size, images are not clickable in general, you can right-click on them and open in a new tab